Move all crates.io instructions to crates.io ops guide

carols10cents opened the pull request:
And merge docs.rs instructions into the docs.rs page.

The crates.io ops guide is [over here](https://github.com/rust-lang/crates-io-ops-guide); you may or may not have access to it, but I have a [pr open there](https://github.com/rust-lang/crates-io-ops-guide/pull/34) adding the information that's removed here.

r? @pietroalbini

carols10cents committed:
Move crates.io removal instructions to crates.io ops guide

Merge docs.rs instructions into the docs.rs page.

Turbo87 commented on src/crates-io/crate-removal.md:
```diff
@@ -1,48 +0,0 @@
-# Crate removal procedure
-
```
maybe keep a link to the moved instructions here for people that are used to looking in this place? (not sure if there are any 😅)

jyn514 replied:
What's the link to the moved instructions?

jtgeibel reviewed (changes requested):
I think we just need to drop the link from `src/SUMMARY.md` and this looks good to me.

carols10cents committed:
Remove obsolete crates.io database maintenance page

The instructions in the crates.io ops guide are newer.

carols10cents committed:
Point to the crates.io ops guide on the crates.io page

carols10cents commented:
I noticed today that the crates.io database maintenance instructions here are older and duplicated with the instructions in the crates.io ops guide, so I removed that page here too. I changed the top-level crates.io page to point to the crates.io ops guide.

jyn514 commented on src/docs-rs/maintenance.md:
```diff
@@ -122,9 +122,18 @@ UPDATE queue SET attempt = 0 WHERE attempt >= 5 AND build_time > 'YYYY-MM-DD HH:
+of a crate (for example after receiving a DMCA), by running a CLI command. The
+people who currently have permissions to access the server and run it are:
+
+* docs.rs Team:
+  * [@pietroalbini](https://github.com/pietroalbini)
+  * [@jyn514](https://github.com/jyn514)
+* Infrastructure Team:
+  * [@Mark-Simulacrum](https://github.com/Mark-Simulacrum)
```
```suggestion
* docs.rs Team:
  * [@jyn514](https://github.com/jyn514)
  * [@syphar](https://github.com/syphar)
  * [@Nemo157](https://github.com/nemo157)
* Infrastructure Team:
  * [@Mark-Simulacrum](https://github.com/Mark-Simulacrum)
  * [@pietroalbini](https://github.com/pietroalbini)
```
I worry that it doesn't make sense to hard-code this separately from https://github.com/orgs/rust-lang/teams/docs-rs ? Can we link there instead?

jyn514 commented on src/docs-rs/maintenance.md:
````diff
@@ -122,9 +122,18 @@ UPDATE queue SET attempt = 0 WHERE attempt >= 5 AND build_time > 'YYYY-MM-DD HH:
+If you are one of these people, log into the server and run:
 
 ```
 cratesfyi database delete-crate CRATE_NAME
````
```suggestion
cratesfyi database delete crate CRATE_NAME
```
There is also a `delete version` command if you only want to delete a specific version.

ehuss commented:
@carols10cents Just checking if you are still interested in working on this?

carols10cents commented:
sure, sorry, lost track of it!
